src/xrGame/GametaskManager.cpp

```diff
@@ -269,6 +270,7 @@ void CGameTaskManager::SetActiveTask(CGameTask* task)
         g_active_task_id[type] = task->m_ID;
         m_flags.set(eChanged, TRUE);
         task->m_read = true;
+        g_DiscordRPC.SetTask(StringTable().translate(task ? task->m_Title : "st_no_active_task").c_str());
```

Comment:
I think, I'd like to move `StringTable().translate()` call inside `SetTask` and `Update` methods. My point is that if anything will be changed in the logic of getting localization, then we'll fix it only inside `DiscordRPC` class.
@Xottab-DUTY what do you think?

Reply:
I totally disagree.

Reply:
We should keep Discord integration abstract from localization system.